timepicker.d.ts

```diff
@@ -0,0 +1 @@
+declare module "react-time-picker/dist/entry.nostyle";
```

Comment:
can we move this into the camel-case.d.ts? If it still works, can we rename that file to something like `module-declarations.d.ts`